sass/module/_availability.scss

```diff
@@ -13,6 +13,7 @@
   color: darken($availability-reservable, 10%);
 }
 
+#user-bookmarks .reserve-button,
```

Comment:
Would  `.material-item .action-button.reserve-button` be an option?

It would ensure consistency whereeever the `material_item.tpl.php` is used.